scripts/brofish-ping.sh

```diff
@@ -11,6 +11,8 @@ TOTAL_RETRIES=3
 SLEEP_TIMEOUT=10
 CPU_THRESHOLD=80
 
+[[ -n "$FW_ZEEK_CPU_THRESHOLD" ]] && CPU_THRESHOLD=$FW_ZEEK_CPU_THRESHOLD
+
```

Comment:
Since we have a default value for CPU_THRESHOLD, this can be simplied as

`CPU_THRESHOLD=${FW_ZEEK_CPU_THRESHOLD:-80}`